src/tests/fixtures/interfaces/InterfaceMergedIntoNode.invalid.ts.expected

```diff
@@ -0,0 +1,24 @@
+-----------------
+INPUT
+----------------- 
+// Note: Node is a built in class.
+/** @gqlInterface */
+interface Node {
+  /** @gqlField */
+  id: string;
+}
+
+-----------------
+OUTPUT
+-----------------
+src/tests/fixtures/interfaces/InterfaceMergedIntoNode.invalid.ts:3:11 - error: Unexpected merged interface. If an interface is declared multiple times in a scope, TypeScript merges them. To avoid ambiguity Grats does not support using merged interfaces as GraphQL interfaces. Consider using a unique name for your TypeScript interface and renaming it.
+
+ Learn more: https://grats.capt.dev/docs/docblock-tags/interfaces/#merged-interfaces
+
+3 interface Node {
+            ~~~~
+
+  node_modules/.pnpm/typescript@5.0.2/node_modules/typescript/lib/lib.dom.d.ts:10247:11
```

Comment:
Having this path (with TypeScript version number) may cause us problems in the future if we want to test against many versions of TS.